media-libs/amdvlk/amdvlk-2020.1.2-r4.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+inherit python-r1 git-r3 multilib-minimal check-reqs
```

Comment:
Do not use the git eclass, instead download the commit's tarball in the way I've shown you.

Reply:
You're genius, thanks